src/main/java/org/opensrp/domain/rapidpro/converter/BaseRapidProClientConverter.java

```diff
@@ -0,0 +1,87 @@
+package org.opensrp.domain.rapidpro.converter;
+
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.DateTime;
+import org.opensrp.domain.IdentifierSource;
+import org.opensrp.domain.rapidpro.contact.zeir.RapidProContact;
+import org.opensrp.domain.rapidpro.contact.zeir.RapidProFields;
+import org.opensrp.service.IdentifierSourceService;
+import org.opensrp.service.UniqueIdentifierService;
+import org.opensrp.util.constants.RapidProConstants;
+import org.smartregister.domain.Client;
+
+import java.util.List;
+
+public abstract class BaseRapidProClientConverter implements RapidProContactClientConverter {
+
+	protected IdentifierSourceService identifierSourceService;
+
+	protected UniqueIdentifierService uniqueIdentifierService;
+
+	public BaseRapidProClientConverter() {
+	}
+
+	public BaseRapidProClientConverter(IdentifierSourceService identifierSourceService,
+			UniqueIdentifierService uniqueIdentifierService) {
+		this.identifierSourceService = identifierSourceService;
+		this.uniqueIdentifierService = uniqueIdentifierService;
+	}
+
+	protected void addCommonZeirProperties(RapidProContact rapidProContact, Client client) {
+		RapidProFields fields = rapidProContact.getFields();
+		client.addAttribute(RapidProConstants.IS_REGISTRATION_COMPLETE, false);
+		client.addAttribute(RapidProConstants.SYSTEM_OF_REGISTRATION, RapidProConstants.MVACC);
+		client.setDateCreated(DateTime.now());
+		if (StringUtils.isNoneBlank(rapidProContact.getName())) {
```

Comment:
```suggestion
		if (StringUtils.isNotBlank(rapidProContact.getName())) {
```

Reply:
`StringUtils.isNoneBlank` is used also to exclude any white blank spaces